python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md

```diff
@@ -81,3 +81,62 @@ Inference time: xxxx s
 --------------------------------------------------------------------------------
 done
 ```
+
+## Example 2: Predict Tokens using `generate()` API using multi processes
+In the example [llama2.py](./llama2.py), we show an experimental support for a Llama2 model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimization and fused decoderlayer optimization on Intel NPUs.
+### 1. Install
+#### 1.1 Installation on Windows
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.10
+conda activate llm
+
+# install ipex-llm with 'all' option
+pip install --pre --upgrade ipex-llm[all]
+pip install --pre --upgrade bigdl-core-npu
+
+pip install transformers==4.40
```

Comment:
I am experimenting on `transformers==3.39.3`, have we verified 4.40?

Reply:
> I am experimenting on `transformers==3.39.3`, have we verified 4.40?

yes, I have also verified `transformers==4.40`. 

Have added `optimize_llm_post` to wrap `lm_head` related logic etc. And considering we may need additional process to address prefill, maybe we could keep separating npu and gpu PP temporarilly.